core/include/cubos/core/al/audio_device.hpp

```diff
@@ -9,6 +9,7 @@
 #include <vector>
 
 #include <glm/glm.hpp>
+#include <miniaudio.h>
```

Comment:
This file/class is meant to abstract the miniaudio library entirely, so there should be no references at all to that library here.